spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -699,6 +699,21 @@ public void setConsumersPerQueue(Integer consumersPerQueue) {
 		 */
 		private Long replyTimeout;
 
+		/**
+		 * The name of the default exchange to use for send operations.
+		 */
+		private String exchange;
+
+		/**
+		 * The value of a default routing key to use for send operations.
+		 */
+		private String routingKey;
+
+		/**
+		 * Enable transactional channels.
+		 */
+		private boolean channelTransacted;
```

Comment:
This one looks totally unrelated to the the first two and makes me a bit neverous. 

@garyrussell, any advice here? Does that sound safe to change the default via properties?

Reply:
Routing key and exchange should default to `null` and make this a `Boolean` - see my other comment.

Reply:
I will leave this as `boolean` and `false` then since the value is tested.